python/ray/__init__.py

```diff
@@ -30,6 +30,9 @@
     os.path.abspath(os.path.dirname(__file__)), "thirdparty_files")
 sys.path.insert(0, thirdparty_files)
 
+import ray.compat  # noqa: E402
+ray.compat.patch_redis_empty_recv()
```

Comment:
Can we actually put the windows check here? And then an assert inside of the patch function. That would make it clear both from here that we're only patching windows and from in the function that it should only be called for windows.

Reply:
Okay, just did.